### DISCUSSION TITLE
My student certification has been passed, but the official didn't send me an email, and I can't use copilot

### DISCUSSION THREAD

**Discussion by caplist** [community/community#65063] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/community/community/assets/54023455/26e7ae92-f443-4cc1-92ed-5dcb9444fa28)
github still needs to pay when setting copilot, what should I do

**Comment by advayc** [comment, marked as answer]
The benefits will be active about 72 hours after your application. You'll get a email along with the PRO badge on your github profile.

**Reply by caplist** [reply]
Thanks!

**Comment by r00ott** [comment]
need to wait like 3 to 4 days
